src/main/java/crazypants/enderzoo/item/ItemGuardiansBow.java

```diff
@@ -110,15 +110,15 @@ public void onPlayerStoppedUsing(ItemStack stack, World worldIn, EntityLivingBas
         
         entityarrow.setDamage(entityarrow.getDamage() + damageBonus + 20);
         
-        worldIn.spawnEntityInWorld(entityarrow);
+        worldIn.spawnEntity(entityarrow);
       }
 
       worldIn.playSound((EntityPlayer) null, entityplayer.posX, entityplayer.posY, entityplayer.posZ, SoundEvents.ENTITY_ARROW_SHOOT, SoundCategory.NEUTRAL,
           1.0F, 1.0F / (itemRand.nextFloat() * 0.4F + 1.2F) + drawRatio * 0.5F);
 
       if (!arrowIsInfinite) {
-        --itemstack.stackSize;
-        if (itemstack.stackSize == 0) {
+        itemstack.shrink(1);
+        if (itemstack.getCount() == 0) {
           entityplayer.inventory.deleteStack(itemstack);
```

Comment:
I will double check tomorrow. isEmpty checks for the stack size == zero as well as a few other things.  I could replace the count==0 to isempty call. this is from source

`    {
        return this == EMPTY ? true : (getItemRaw() != null && getItemRaw() != Item.getItemFromBlock(Blocks.AIR) ? (this.stackSize <= 0 ? true : this.itemDamage < -32768 || this.itemDamage > 65535) : true);
    }`